data-discovery/use-cases.md

```diff
@@ -2,12 +2,19 @@
 
 The following use cases must be considered when designing a mechanism for a data pod to define how its data is stored and indexed so that a given application can automatically find the data that it needs.
 
-## Identification scheme
+## New application
 
-In order for data of a particular type to be created,
-we need to tell an application what the prefered mechanism for creating
-instance ids should be.
+Alice used `AncientBookmarks` application from `OldBookmarks.com`, to record a number of bookmarks which are stored on her POD.  Recently Alice's friend Bob recommended that she upgrade to the application `MyThoughtsOnIt` from `WebAnnontaions.com`.
```

Comment:
Conformance to [RFC2606](https://tools.ietf.org/html/rfc2606#section-2) suggests that `OldBookmarks.com` and `WebAnnontaions.com` be changed to `OldBookmarks.example` and `WebAnnontaions.example`, respectively, here and subsequently in this document.